java/java-guestbook/backend/src/main/java/cloudcode/guestbook/backend/BackendApplication.java

```diff
@@ -0,0 +1,20 @@
+package cloudcode.guestbook.backend;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class BackendApplication {
+
+    public static void main(String[] args) {
+        for (String expectedVar : new String[]{"PORT", "GUESTBOOK_DB_ADDR"}) {
```

Comment:
probably ok -- hopefully the optimizer will catch this, but seems like a const String[] would be better.

Reply:
fixed